controllers/vrg_volrep.go

```diff
@@ -1250,13 +1255,13 @@ func (v *VRGInstance) selectVolumeReplicationClass(
 
 	storageClass, err := v.getStorageClass(namespacedName)
 	if err != nil {
```

Comment:
(related) We can add the check for `v.vrcUpdated` inside the function `updateReplicationClassList` and fix the invocation from this function as well `updatePVCList` to reduce cognitive complexity of this function.